app/views/configs/_config.html.erb

```diff
@@ -6,14 +6,14 @@
                                                           .try(:join, ', '),
                                 help: config.help_text,
                                 id: "config_options_#{config.config_key}" %>
-      <%= f.submit "Update options for #{config.config_key.to_s.humanize}", class: 'btn btn-primary' %>
+      <%= f.submit t('configs.config.update_options_for', option: config.config_key.to_s.humanize), class: 'btn btn-primary' %>
```

Comment:
does humanize work okay with spanish? I know @lomky had some problems with the pluralizer.